packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -696,6 +628,88 @@ export default class GoogleCalendarService implements Calendar {
       throw error;
     }
   }
+
+  async watchCalendar({ calendarId }: { calendarId: string }) {
+    const calendar = await this.authedCalendar();
+    const res = await calendar.events.watch({
+      // Calendar identifier. To retrieve calendar IDs call the calendarList.list method. If you want to access the primary calendar of the currently logged in user, use the "primary" keyword.
+      calendarId,
+      requestBody: {
+        // A UUID or similar unique string that identifies this channel.
+        id: uuid(),
+        type: "web_hook",
+        address: `${WEBHOOK_URL}/api/integrations/googlecalendar/webhook`,
+        token: process.env.CRON_API_KEY,
```

Comment:
Same as previous comment. I think we should make a separate secret for this to decouple it from the cron jobs.